apps/balancer-tools/seed.ts

```diff
@@ -523,83 +592,98 @@ DO UPDATE SET chain_id = EXCLUDED.chain_id, updated_at = NOW();
 `);
 }
 
-// async function fetchTokenPrices() {
```

Comment:
removed this for now as this has to be refactor to work properly

Reply:
if it has to be refactored, why remove it? Is not of that usable? 😅